src/profile/vcpr.jl

```diff
@@ -0,0 +1,74 @@
+
+function profilevc(m::LinearMixedModel{T}, val::T, rowj::AbstractVector{T}) where {T}
+    optsum = m.optsum
+    function obj(x, g)
+        isempty(g) || throw(ArgumentError("g must be empty"))
+        updateL!(setθ!(m, x))
+        optsum.sigma = val / norm(rowj)
+        objctv = objective(m)
+        return objctv
+    end
+    opt = Opt(optsum)
+    NLopt.min_objective!(opt, obj)
+    fmin, xmin, ret = NLopt.optimize!(opt, copyto!(optsum.final, optsum.initial))
+    _check_nlopt_return(ret)
+    return fmin, xmin
+end
+
+function profileσs!(val::NamedTuple, tc::TableColumns{T}; nzlb=1.0e-8) where {T}
```

Comment:
```suggestion
"""
     profileσs!(val::NamedTuple, tc::TableColumns{T}; nzlb=1.0e-8) where {T}
     
Profile the variance components.

!!! note
   This method is called by `profile` and currently considered internal.
   As such, it may change or disappear in a future release without being considered breaking.
"""
function profileσs!(val::NamedTuple, tc::TableColumns{T}; nzlb=1.0e-8) where {T}
```